internal/utils/aggregator.go

```diff
@@ -104,17 +101,8 @@ func filter_valid_csv_records(main_df dataframe.DataFrame) (dataframe.DataFrame,
 		dataframe.F{Colname: "owner_name", Comparator: series.Neq, Comparando: ""},
 		dataframe.F{Colname: "owner_kind", Comparator: series.Neq, Comparando: "<none>"},
 		dataframe.F{Colname: "owner_name", Comparator: series.Neq, Comparando: "<none>"},
-	)
-
-	no_of_dropped_records := main_df.Nrow() - df.Nrow()
-
-	return df, no_of_dropped_records
-}
-
-func filter_valid_k8s_object_types(df dataframe.DataFrame) dataframe.DataFrame {
-	return df.Filter(
 		dataframe.F{
-			Colname:    "k8s_object_type",
+			Colname:    "workload_type",
```

Comment:
Any specific reason we changed the colname from `k8s_object_type` to `workload_type`? 
As far as I remember `workload_type` does not tell the correct representation of workload. For example - even if the workload is of type replicaset, `workload_type` field is set to `deployment`.  Due to this behaviour we have a function `determine_k8s_object_type` which determines the correct type of workload and sets it in `k8s_object_type` column.

Reply:
Actually let me explain more maybe,
The failure was happening because there were no rows which had valid `k8s_object_type` or I can say it was empty. However we did not had `df.Nrows==0` after filtering the valid `k8s_object_type` and so we ended up giving empty df to Groups further.
It looked we can just check `df.Nrows==0` after `filter_valid_k8s_object_types` and that should fix it, and it can. However then it also felt we can drop the empty and non valid rows in `filter_valid_csv_records` based on the workload_type, and then do the `determine_k8s_object_type` . So indeed we do not need to do `filter_valid_k8s_object_types` after `determine_k8s_object_type`?

Reply:
Got your point. I agree we can apply filter before calling `determine_k8s_object_type`. But my concern is that you have removed the `filter_valid_k8s_object_types` function which actually filters workloads based on there workload_type. So in this case workload of type `job` will also get pass.

Reply:
Instead of comparing actual workload type, can we simply add here `dataframe.F{Colname: "workload_type", Comparator: series.Neq, Comparando: ""},`

Reply:
and keep `filter_valid_k8s_object_types` function as it is.